core/jvm/src/test/scala/fs2/TaskSpec.scala

```diff
@@ -0,0 +1,23 @@
+package fs2
+
+import scala.concurrent.duration._
+
+class TaskSpec extends Fs2Spec{
+
+  "Task" - {
+    "Ref" - {
+      "Set increments nonce" in {
```

Comment:
I think nonce is here internal implementation detail. What we really solving here is interleaving set and modify// access. I would prefer updated name with perhaps a short comment explaining what this tests verifies.